.prettierrc

```diff
@@ -1,5 +1,8 @@
 # Prettier configuration
 
+plugins:
+  - prettier-plugin-java
```

Comment:
@zhx828 I was having this [issue](https://github.com/jhipster/prettier-java/issues/508) where prettier couldn't figure out which parser to use for java files so I forced it to use the java parser.